lib/stitches/cache_injector.rb

```diff
@@ -0,0 +1,32 @@
+require 'lru_redux'
+
+module Stitches::CacheInjector
+  MAX_ITEMS_IN_CACHE = 100
+  CACHE_DURATION_IN_SECONDS = 5 * 60 # five minutes
```

Comment:
❓ Do you anticipate consuming apps would need to customize these in any way?

Reply:
Do you mean apps that consume the gem? I don't see a current need, but I'm happy to make this a little more configurable. This is the second comment in this regard so I might just go ahead and do that.